rc/tools/ctags.kak

```diff
@@ -25,7 +25,7 @@ define-command -params ..1 \
                 cut -f 1 "$tags" | grep -v '^!' | uniq > "$namecache"
             fi
             cat "$namecache"
-        done} \
+        done | sort } \
```

Comment:
Piping into sort means we wont get partial results in shell-script-candidates, which is a shame in the cases such as doc or man where we could get candidates progressively out of the find command. I wonder if sorting really is better than random order.